### PR TITLE
phantom2 needs newline after address (fixes #212)

### PR DESCRIPTION
**Pull request by fmachella**
it solve phantomjs 2.1.1 issue about newline.

Fixes #212

### CODE REVIEW

**Comment by searls** [comment]
can you please write "fixes #212" in your PRs so they're referenced and automatically close when a PR is merged?

**Comment by fmachella** [comment]
Is it good for you?

**Comment by searls** [comment]
I'm just sharing a protip about how github automatically parses. When the summary of the PR text includes "fixes #xxx" or "resolves #xxx", then issue #xxx will automatically be linked & then (when merged) automatically closed.

File: lib/assets/javascripts/jasmine-runner.js [hunk 1]
```diff
@@ -90,6 +90,10 @@
 
   var address = args[1];
   console.log('Running: ' + address);
+  if(phantom.version.major===2){
```
Comment (searls) [inline lib/assets/javascripts/jasmine-runner.js]: Could you please format this in standardjs style?

```js
if (phantom.version.major === 2) {
  address += '\n'
}
```

Comment (fmachella) [reply lib/assets/javascripts/jasmine-runner.js]: done

**Comment by searls** [comment]
Landed in 0.14.5

**Pull request merged**